tests/Element/TagBlockTest.php

```
@@ -116,6 +117,38 @@ protected function run(): string
     }
 
+    public function testEndRemovesEmptyContextStackAfterBalancedRender(): void
+    {
+        $stackProperty = new ReflectionProperty(BaseBlock::class, 'stack');
+        $mainThreadProperty = new ReflectionProperty(BaseBlock::class, 'mainThread');
+
+        $stackProperty->setValue(null, null);
+        $mainThreadProperty->setValue(null, null);
+
+        TagBlock::tag()->begin();
+        TagBlock::end();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Core\Tests\Support\Stub\TagBlock' in method 'testEndRemovesEmptyContextStackAfterBalancedRender'. (undefined)

(StaticAccess)